test/parallel/test-http-exceptions.js

```diff
@@ -21,7 +21,10 @@
 
 'use strict';
 require('../common');
+const Countdown = require('../common/countdown');
 const http = require('http');
+const NUMBER_OF_EXCEPTIONS = 4;
+const countdown = new Countdown(NUMBER_OF_EXCEPTIONS, () => process.exit(0));
```

Comment:
This should be wrapped in `common.mustCall`, also should the listener of `process.on('uncaughtException')` and `server.listen`

This kind of reminds me: should we wrap the cb in `common.mustCall` by default? @jasnell

Reply:
> should we wrap the cb in `common.mustCall` in the countdown module by default?

Yeah, I've been thinking the same thing. It would be worthwhile I think

Reply:
Come to think of it, maybe this test does not need the countdown module and can just wrap the `process.on('uncaughtException')` in `common.mustCall(..., NUMBER_OF_EXCEPTIONS)`

Reply:
@joyeecheung unless i am overlooking something; I didn't wrap it in a `mustCall` since not invoking the callback will result in a timeout and hence a failure in the test anyways, but adding it does not hurt as well.

Reply:
If you do go with this (and I'm with @joyeecheung that this maybe doesn't need `Countdown`) Can you please wrap `process.exit(0)` in `{` and `}`?

Without `{` and `}`, then the result of `process.exit(0)` is the return value of the function. That function shouldn't have a return value.

Reply:
@Trott but `process.exit()` terminates the process synchronously, so the return value will not get passed back from the invocation, so it does not really matter what you return.

Reply:
Regarding wrapping the countdown with a mustCall, i think its a good idea to decide soon before contributors start implementing more refactors

Reply:
I think in general, use `common.mustCall` if the code calling `countdown.dec()` is always the same, and use `countdown` if the caller would be different. That way the code is cleaner. If we are trying to make the code cleaner by using `countdown`, then I see no reason not to make it even cleaner with `common.mustCall`

Reply:
Also...can we implement `common.mustCall` on top of `countdown`?

Reply:
@joyeecheung i have implemented `mustCall` on top of countdown, but im not sure i fully understand your suggestion about replacing `countdown` with `mustCall`

```javascript
process.on('uncaughtException', common.mustCall(onUncaughtException, NUMBER_OF_EXCEPTIONS));
```

however how are we going to tell the code to `process.exit(0)`? i cant see happening without a counter